.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  ci-go: ipfs/ci-go@0.2.9
+  ci-go: ipfs/ci-go@0.3.1
```

Comment:
This was added to see if it helped at all, it didn't really. We can remove this if we're just going to land the other CI PR #712